packages/bigquery/src/dataset.js

```diff
@@ -345,6 +326,32 @@ Dataset.prototype.getTables = function(query, callback) {
 };
 
 /**
+ * List all or some of the tables in your project as a readable object stream.
+ *
+ * @param {object=} query - Configuration object. See
+ *     {module:bigquery/dataset#getTables} for a complete list of options.
+ * @return {stream}
+ *
+ * @example
+ * dataset.getTableStream()
+ *   .on('error', console.error)
+ *   .on('data', function(table) {})
+ *   .on('end', function() {
+ *     // All tables have been retrieved
+ *   });
+ *
+ * //-
+ * // If you anticipate many results, you can end a stream early to prevent
+ * // unnecessary processing and API requests.
+ * //-
+ * dataset.getTableStream()
+ *   .on('data', function(table) {
+ *     this.end();
+ *   });
+ */
+Dataset.prototype.getTableStream = common.paginator.streamify('getTables');
```

Comment:
Looks like there's a way to describe it with multiple tags.. but I think that would require a change to both the docs site and our doc scripts. So if you're cool with the sentence I'll go with that :)

http://usejsdoc.org/tags-event.html